addons/mail/static/tests/discuss/core/new_message_separator.test.js

```diff
@@ -0,0 +1,222 @@
+import { presenceService } from "@bus/services/presence_service";
+import { rpcWithEnv } from "@mail/utils/common/misc";
+import {
+    assertSteps,
+    click,
+    contains,
+    defineMailModels,
+    insertText,
+    onRpcBefore,
+    openDiscuss,
+    openFormView,
+    start,
+    startServer,
+    step,
+    triggerHotkey,
+} from "@mail/../tests/mail_test_helpers";
+import { describe, test } from "@odoo/hoot";
+import { withUser } from "@web/../tests/_framework/mock_server/mock_server";
+import { Command, mockService, serverState } from "@web/../tests/web_test_helpers";
+import { mockDate } from "@odoo/hoot-mock";
+
+/** @type {ReturnType<import("@mail/utils/common/misc").rpcWithEnv>} */
+let rpc;
+describe.current.tags("desktop");
+defineMailModels();
+
+test("remove after going to the thread twice", async () => {
```

Comment:
Oka-ish wording. I would suggest this as easier to read & grasp functional purpose: _"persists until close then open thread again"_

Reply:
It's more about navigating to the thread. I don't even know what close is supposed to mean in this context, there is no way to "close" a thread on the discuss app. I could understand when it comes to chat windows though. Also I think it might convey the wrong idea since it suggests the separator should disappear the first time we go back while its not the case?